commands/post_e2e_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mmctl/v6/client"
+	"github.com/mattermost/mmctl/v6/printer"
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlE2ETestSuite) TestPostListCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("List all posts for a channel", func(c client.Client) {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+		cmd.Flags().Int("number", 1, "")
+		err := postListCmdF(c, cmd, []string{s.th.BasicTeam.Name + ":" + s.th.BasicChannel.Name})
+		s.Require().Nil(err)
+		s.Equal(1, len(printer.GetLines()))
```

Comment:
I'd rather create a new channel and post some messages then assert those to test this behavior.

Reply:
Update e2e-tests